chainer_chemistry/functions/mean_squared_error.py

```diff
@@ -0,0 +1,75 @@
+import numpy
+
+from chainer import cuda
+from chainer import function_node
+import chainer.functions
+from chainer.utils import type_check
+
+
+class MeanSquaredError(function_node.FunctionNode):
+
+    """Mean squared error (a.k.a. Euclidean loss) function."""
+
+    def __init__(self, ignore_nan=False):
+        self.task_weight = None
```

Comment:
Can you write 
`#TODO (mottodora): implement task weight calculation`
so that others can understand it is simply not implemented yet?